salt/client/ssh/client.py

```diff
@@ -196,3 +197,41 @@ def cmd_async(self, low, timeout=None):
         '''
         # TODO Not implemented
         raise SaltClientError
+
+    def cmd_subset(
```

Comment:
It's kind of unclear how this is useful to me. You get a command run on a random set of minions but then you get the results of the function being run on those minions and no clear indication of how to target the _remaining_ minions in the group you're trying to hit, since you don't get any of that data back.